### PR TITLE
Teach eshost the `--edit` option: allows modifying `--args` or `--tags` of a host

### PR DESCRIPTION
**Pull request by dilijev**

### CODE REVIEW

**Comment by dilijev** [comment]
Putting this up now. I'll add tests before merging.

File: bin/eshost.js [hunk 1]
```diff
@@ -193,11 +195,19 @@ if (argv.add) {
...
+
+// edit a host
+if (argv.edit) {
+  hostManager.edit(config, argv.edit, argv.args, hostTags);
+  console.log(`Host '${argv.edit}' edited`);
+  process.exit(0);
+}
+
+if (argv.args) {
+  // at this point in execution, implies (argv.args && !(argv.add || argv.edit))
```
Comment (bterlson) [inline bin/eshost.js]: I think this would be more clear as a check on line 190 with the contents of this comment.

**Comment by bterlson** [comment]
Looks good. I may continue hand-editing the JSON but this seems needed.

File: bin/eshost.js (hunk 1 above)
Comment (dilijev) [inline bin/eshost.js]: I see what you mean (approximately, I think), this whole block of `if (argv.add)` ... `if (argv.edit)` ... `if (argv.args) // fail` enclosed in `if (argv.add || argv.edit) { ...  }`

**Pull request merged**